libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.html

```diff
@@ -1,6 +1,11 @@
 <div id="emptyDiv" *ngIf="isZero" class="arrow-variation zero" [ngClass]="size"></div>
 <div *ngIf="!isZero" class="arrow" [ngClass]="size">
-  <svg width="100%" height="100%" viewBox="0 0 24 24" style="display: block">
-    <path [attr.fill]="arrowColor" [attr.d]="arrowDirection" />
-  </svg>
+  <ng-container *ngIf="!pdfPrint; else pngArrow">
```

Comment:
Instead of performing a NOT operation in the template, you could change the name of the variable and the content of the ng-container for the content of the ng-template and write `*ngIf="printing; else svgTemplate"`.